src/utils/account.js

```diff
@@ -0,0 +1,28 @@
+import { firebase, DB } from "db/firebase";
+import { GetWeeklyBaseFromTime } from "./date";
+const GetSemesterData = async (id, semester) => {
+  let data = { value: [], ids: [] };
```

Comment:
Using `value` as a field may be too generic. Is there a way to better describe the content?

Reply:
The naming is because the data we get from firebase look like {value:[], ids:[]}